### PR TITLE
Create shortcut to undo back to main branch

### LINKED ISSUE #324: [feature request] back to the main branch.
currently we have 

> "[b] Go back to the previous branching point."

which is great ,can we have another keymap such as Shift+B back to the main branching point.

in order to deal with multi branches ,usually willl press b several times before return to the main branch.

### PR DESCRIPTION
**Pull request by LiamHz**
Added "Shift-b" as a keyboard shortcut to undo back to the main branch

Closes #324

### CODE REVIEW

File: katrain/core/game.py [hunk 1]
```diff
@@ -297,6 +302,10 @@ def undo(self, n_times=1, stop_on_mistake=None):
                 cn = cn.parent
             if break_on_branch and len(cn.children) > 1:
                 break
+            elif break_on_main_branch and len(cn.children) > 1:
+                last_branching_node = cn
+        if break_on_main_branch:
```
Comment (sanderland) [inline katrain/core/game.py:307]: very clever, I like it!

Comment (sanderland) [reply katrain/core/game.py:307]: Actually too clever, it doesn't work ;)

Comment (sanderland) [reply katrain/core/game.py:307]: fixed in faa5af5

Comment (LiamHz) [reply katrain/core/game.py:307]: ahh, good catch - my version went to the earliest branching node in the main branch. I'll try to tackle a few more of the GitHub issues over the next week or so - hopefully my future PRs will be better :)

**Pull request merged**